rosdep/base.yaml

```
@@ -2589,6 +2589,31 @@ libboost-chrono-dev:
   opensuse: [libboost_chrono1_66_0-devel]
   rhel: [boost-devel]
   ubuntu: [libboost-chrono-dev]
+libboost-coroutine:
+  debian:
+    bullseye: [libboost-coroutine1.74.0]
+    buster: [libboost-coroutine1.67.0]
+  fedora: [boost-coroutine]
+  gentoo: [dev-libs/boost]
+  nixos: [boost]
+  openembedded: [boost@openembedded-core]
+  opensuse: [libboost_coroutine1_66_0]
+  rhel: [boost-coroutine]
```

```suggestion
  rhel:
    '*': [boost-coroutine]
    '7': null
```

Nulling out RHEL 7 as caught by the linters.